enterprise/frontend/src/embedding-sdk/components/private/SdkQuestion/components/EditorButton/EditorButton.stories.tsx

```diff
@@ -4,7 +4,7 @@ import { Box } from "metabase/ui";
 
 import {
   EditorButton,
-  type InteractiveQuestionEditorButtonProps,
+  type EditorButtonProps as EditorButtonProps,
```

Comment:
I think the editor rename went a bit wrong here.

```suggestion
  type EditorButtonProps,
```